2017/kickstart.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: news
+title: 2017 Kickstart
+permalink: /2017/kickstart
+date: 2016-10-01
```

Comment:
What's this the date of? As far as I can tell this isn't a news article, so it's not a publishing date, and it's definitely not the date of the event.

Please remove if redundant.